Automattic/wp-calypso
# Site Selector: Remove search debounce for better UX

Related https://github.com/Automattic/wp-calypso/pull/73335

## Proposed Changes

Removes the debounce from Site Selector search so new results appear immediately, instead of after a delay.

## Testing Instructions

1. Navigate to My Home and open up the Site Selector.
2. Enter a search in the Site Selector and observe the performance boost.
